lib/main.dart

```diff
@@ -154,15 +153,14 @@ class _MyHomePageState extends State<MyHomePage> {
         /// For each workout in the returned DB data snapshot.
         ///
         for (final workout in snapshot.data)
+          /// This is a GestureDetector widget that is used to handle different types of touch events.
```

Comment:
Please remove the comment referencing the gesture detector as it is no longer being used